test/unittest/blas2/blas2_trsv_test.cpp

```diff
@@ -0,0 +1,128 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename blas2_trsv_test.cpp
+ *
+ **************************************************************************/
+
+#include "blas_test.hpp"
+
+template <typename T>
+using combination_t = std::tuple<int, bool, bool, bool, int, int>;
```

Comment:
`int` can be replaced with `index_t` here.

Reply:
All the tests in `test/unittest/blas?/*.cpp` utilize `int` in place of `index_t`.

Reply:
Yes, but this was pointed out in the `spr` PR, and so I thought we could start using `index_t` from now onwards. We'll probably need to spend some time to make the code consistent later on.